counter/app.js

```diff
@@ -0,0 +1,30 @@
+const value = document.querySelector('#value');
+const btns = document.querySelectorAll('.btn');
+
+let counter = 0;
+
+const counterColor = () => {
+  if (counter > 0) {
+    value.style.color = 'green';
+  } else if (counter === 0) {
+    value.style.color = '#222';
+  } else {
+    value.style.color = 'red';
+  }
+};
```

Comment:
hi @tsheporamantso 
Great work on creating a function that will change colors and invoke it on click events 👍